gen/definitions/port_group.yaml

```diff
@@ -0,0 +1,47 @@
+---
+name: PORT Group
+rest_endpoint: /api/fmc_config/v1/domain/{DOMAIN_UUID}/object/portobjectgroups
+data_source_name_query: true
+doc_category: Objects
+attributes:
+  - model_name: name
+    type: String
+    description: User-created name of the object.
+    mandatory: true
+    example: portgroup_obj1
+  - model_name: type
+    type: String
+    mandatory: true
+    example: "PortObjectGroup"
+  - model_name: description
+    type: String
+    description: Optional user-created description.
+    example: "My port group description"
+  - model_name: overridable
+    type: Bool
+    description: Indicates whether object values can be overridden.
+    exclude_example: true
+    test_value: "true"        
+  - model_name: objects
+    tf_name: ports
+    type: Set
+    mandatory: true
+    attributes:
+      - model_name: id
+        type: String
+        description: UUID of the port (such as fmc_port.test.id, etc.).
+        id: true
+        example: 0050568A-232D-0ed3-0000-004294971602
+        test_value: fmc_port.test.id
+      - model_name: type
+        type: String
+        mandatory: true
+        example: "ProtocolPortObject"
```

Comment:
Are there other options as well? If so, we should list them using `enum_values` otherwise I would hardcode the value.

Reply:
Done